app/helpers/place/finish_line_helper.rb

```diff
@@ -0,0 +1,29 @@
+module Place
+  module FinishLineHelper
+    MAPS_API_KEY = Rails.application.secrets.maps_api_key
```

Comment:
As `MAPS_API_KEY` is using in multiple places (have a look on layouts) I'd suggest to create a method in `application_helper` like:
```
def maps_api_key
  ENV['MAPS_API_KEY']
end
```
and use it.